src/App.js

```diff
@@ -1,42 +1,134 @@
 import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddTodoForm';
-import TodoListItem from './TodoListItem';
+// import TodoListItem from './TodoListItem';
+// import { json } from 'express';
 
 
 
 function App() {
  
   const [todoList, setTodoList] = React.useState([]);
   const [isLoading, setIsLoading] = React.useState(true);
+
+
+  const fetchData = async() => {
+      const options = {
+        method: "GET", 
+        headers: {
+          Authorization: `Bearer ${process.env.REACT_APP_AIRTABLE_API_TOKEN}`
+        }
+      };
+      const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+      
+      console.log(url);
+      console.log("test",options);
```

Comment:
It's generally a good practice to remove code that's only in place for testing purposes before we merge the branch

Reply:
Fixed